spec/factories/patients.rb

```diff
@@ -30,10 +30,7 @@
                              assigning_user_id: registration_user.id })
     end
     reminder_consent { Patient.reminder_consents[:granted] }
-
-    trait :hypertension do
-      medical_history { build(:medical_history, :hypertension_yes, patient_id: id) }
-    end
```

Comment:
Good call 😄 @prabhanshuguptagit heads up, that the default patient factory is "hypertension_yes" with this change.

Reply:
Is this wrt seed data gen? I checked, that is not being affected (since we use payloads and medical histories are a separate payload). Thanks for the heads up.